src/init-app.ts

```diff
@@ -185,36 +196,33 @@ const setInnerHTMLWithScriptsAndOnLoad = async function (elm, html) {
 
   const allRemoteScripts = [...scriptsWithDefer, ...scriptsWithoutDefer];
 
-  // Try to make the scripts wait until the page had loaded before running
-  const asyncLoadPromises = [];
-
-  // insert async scripts in bulk
-  const fragment = new DocumentFragment();
-  allRemoteScripts.forEach((oldScript) => {
+  // Try to make the scripts load in sequential order
+  const blockLoadingScripts = [];
```

Comment:
This bit was tricky. Before, they were loading in somewhat random order. This led to errors when the user-script loaded before the library script it needed had finished. 

Changing from `Promise.all` to a plain "for" loop enforces the correct execution order.